lib/bolt/cli.rb

```diff
@@ -634,13 +634,25 @@ def list_plans
     end
 
     def list_targets
+      if options[:targets] || options[:rerun] || options[:query]
```

Comment:
We also use these targetting options on line 265 - making those a constant and replacing the references would be good so if they change they're all in one place.